README.md

```diff
@@ -19,6 +19,18 @@ the UI framework used in [Fuse](https://www.fusetools.com/) apps.
 * `build.sh` runs stuff (which downloads uno if needed), and builds all packages.
 * `test.sh` runs all tests.
 
+
+### Fuse
+
+If you want to use a self-built fuselibs installation with an installed Fuse, you can do this through a `.unoconfig`-file, something like this:
```

Comment:
This doesn't mention where to place the .unoconfig file. For instance:

If you want to use a locally built copy of fuselibs with an installed Fuse, create a file named `.unoconfig` in your home directory and add the following: